### ISSUE TITLE
Add references to old blog posts?

### ISSUE THREAD

**Issue by dwieringa** [shageman/component-based-rails-applications-book#6]
I'm finding the book hugely helpful.  I spent a bunch of time trying to get up to speed on this topic through some of the/your old blog posts at Pivotal.  It was only after a few days of slogging and pulling together resources that I discovered it had recently been compiled into the book (and cbra.info).

If possible, consider updating the old blog posts to point people to cbra.info.  Such a time saver!

For what it's worth, I first tried the Kindle trial and found the formatting quite poor/distracting...I assumed the book hadn't been given much attention yet in that area.  In an attempt to see if the EPUB version was better, I tried it at oreilly.com and found the formatting great.  I'll probably end up with the EPUB hoping it matches oreilly.com.  Is this a factor of Kindle limitations or just that the Kindle edition hasn't been given as much love yet?

**Comment by shageman** [comment]
Thanks, @dwieringa for your comments and suggestions! I think I may have lost edit access to some of those Pivotal posts, but will try to get a link to [cbra.info](http://www.cbra.info) in there.

Regarding the difference between the Kindle version and the epub version - I am sorry to hear that. I am glad the epub version works out! I will check with the publisher to see what causes the difference.

Finally, I am glad you are enjoying the book! Please consider leaving me a review on [Amazon](https://www.amazon.com/Component-Based-Rails-Applications-Addison-Wesley-Professional-dp-0134774582/dp/0134774582/) - it really helps get people to take a look and we still need to spread the message to a lot more folks!

Cheers!

**Comment by dwieringa** [comment]
@shageman I've since checked out the sample content in the Kindle app on an iPad.  That's not as bad.  My initial concern was from Amazon's Kindle Cloud Reader in the browser on my Mac.  ...and maybe Amazon's "Look Inside" feature.  It looks like rendering varies.  In the Kindle app on my iPad it was ok, but on orielly.com it's still an improvement (code formatting, etc).

I've now purchased the electronic version from informit.com and notice the different versions from there vary as well (pdf vs epub), so it looks like depends mostly on the reader.

Regardless, the content is great and helpful.  I just posted a review on Amazon
